src/membrane-keypad-element.ts

```diff
@@ -182,4 +187,26 @@ export class MembraneKeypadElement extends LitElement {
       );
     }
   }
+
+  private keyStrokeDown(key: string) {
+    const keyName = `key-${key.toLowerCase()}`;
+    const text = keyName[keyName.length - 1];
+    const selectedKey = this.shadowRoot?.querySelector(`[data-key-name="${keyName}"]`);
+    this.down(text, selectedKey as SVGElement);
+  }
+
+  private keyStrokeUp(key: string) {
+    let selectedKey = this.shadowRoot?.querySelector('.pressed');
+    const keyName = `key-${key.toLowerCase()}`;
+
+    if (key === 'Shift') {
+      this.shiftPressed = false;
+      selectedKey?.classList.remove('pressed');
+      return;
+    }
+
+    const text = keyName[keyName.length - 1];
```

Comment:
hmm doesn't `keyName[keyName.length - 1]` equal `key.toLowerCase()`? 

Also, with the current code, the following happens:
1. If you click on "A" with the mouse, the event is emitted with `{ detail: { key: "A" } }`
2. If you press "A" on the keyboard,  the event is emitted with `{ detail: { key: "a" } }` (note the difference)
3. If you press some other key, that does not exist on the keypad, e.g. the End key, an event is emitted with the last letter of the key name, i.e.  `{ detail: { key: "a" } }`

You can you the "EventLogger" panel of storybook to view the events that are sent for each keypress